src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -305,10 +305,9 @@ private function getResourcesToWatch(ContainerBuilder $container, array $config)
     {
         $paths = array_unique(array_merge($this->getBundlesResourcesPaths($container, $config), $config['mapping']['paths']));
 
-        // Default paths
-        if (!$paths) {
+        if (!$config['mapping']['paths']) {
             $projectDir = $container->getParameter('kernel.project_dir');
-            foreach (["$projectDir/config/api_platform", "$projectDir/src/ApiResource", "$projectDir/src/Document", "$projectDir/src/Entity"] as $dir) {
```

Comment:
`getBundlesResourcesPaths` only concerns bundles. If you remove this, paths `src/Document` and `src/Entity` won't work in a Symfony project. Please rollback this modification.

Reply:
This is working in api-platform/core tests because it uses a TestBundle, not a standard Symfony project architecture